cpp_exercise/src/lib/filehandlerlib.cc

```diff
@@ -34,16 +34,16 @@ void FileHandler::createFile(){
    
     _fd = open(_file_name.c_str(), O_RDWR| O_CREAT | O_TRUNC, _mode);
     if(_fd<0){
-        std::cout<<errno<<std::endl;
+       std::cout<<strerror(errno)<<std::endl;
         throw IPCException("FileHandler ERROR: Cannot Open the File Correctly");
     }
 
 
-    int err = close(_fd);
-     if(err<0){
-         std::cout<<errno<<std::endl;
-        throw IPCException("FileHandler ERROR: Cannot Close the File Correctly");
-    }
+    // int err = close(_fd);
```

Comment:
It will be nicer to not push any commented code. Applies also to some code below

Reply:
Fixed